examples/text_summarization/extractive_summarization_cnndm_transformer.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>1) Would be nice to proof-read the texts once to rephrase sentences like "we suggest you continue with set as True first and experiment with data preprocessing with QUICKRUN set as True."</p><p>2) No need to do it now, but wondering if multiprocessing can speed up the preprocessing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/515/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
let's post the work of multiprocessing.